src/assets/about/content.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "managed-simple-data-exchanger-frontend",
+  "repositoryPath": "https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend",
+  "license": "Apache-2.0",
+  "licensePath": "https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend/blob/2.0.9/LICENSE",
```

Comment:
You should do this in a dynamic way, you won't remember updating the tag for each release. The changelog already describes version 2.0.10 and the links here point to 2.0.9.

Reply:
sure.

Reply:
dynamic values generation scripts added.